targets/commands.go

```diff
@@ -78,7 +78,7 @@ func TelegramAlerting(ops HTTPOptions, cfg *config.Config, c client.Client) {
 // GetEndPointsStatus retsurns status of the configured endpoints i.e, lcd, val and external rpc.
```

Comment:
// GetEndPointsStatus returns status of the configured endpoints i.e, lcd, val and external rpc.